### DISCUSSION TITLE
Add email custom field

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Planner#1564] [Feature Requests]
> I'm using USP PRO to let user post their content on my site, meanwhile I have PLANNER PRO from you.
The issue is that the email address is stored in a custom field (usp-email).
>
> Is there anyway to get planner pro to send notification emails to that address instead of the post author (which would be me)
> 
>
> But then maybe I can request a future feature, since wordpress often is built on different plugins, and if a notification program (yours) can't read an email address from a custom field, in the post, then it can't handle other plugins.
> 
> If a site, like the one i'm building now, is using usp pro (frontend post publish system), that allows everybody to write a post, (and let us preview them, of course, before publishing), then it's necessary to store the visitors mail in a custom field, we can't create a new user for everybody that wants to publish a post.
> So implanting a function, (see screenshot), that allows the admin (us) to use a "custom" field to "send to", would be highly appreciated.
> 

![image](https://github.com/publishpress/PublishPress-Planner/assets/83811723/9aa63823-3488-4b8b-9d9b-8856ddefa30c)

**Comment by Tempis-swe** [comment]
That would be perfekt for me to, since I also use custom fields.
